index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en" dir="ltr">
```

Comment:
- [ ] You should use the same pr for review until you get the approval. In this case you have gotten an reivew on another pull request which i was not able to find it. Now you are using this pull request for the second review. Please don't do this because this is againt the Github Flow and breaks the flow of application development. Always use the same pull request unless you get the approval :blush: